addons/sourcemod/scripting/surftimer/commands.sp

```diff
@@ -366,8 +366,8 @@ public Action Command_ToggleHints(int client, int args)
 public Action Command_PRinfo_help(int client, int args){
 
 	if(IsValidClient(client)){
-		PrintToConsole(client, "%t", "PRinfo_help");
-		CPrintToChat(client, "%t", "PRinfo_help_chat", g_szChatPrefix);
+		PrintToConsole(client, "%T", "PRinfo_help");
```

Comment:
you should read my comment again https://github.com/surftimer/SurfTimer/pull/424#discussion_r876637798
when you're doing this, it would requires the client index behind the phrase name.
in this case it would be:
`PrintToConsole(client, "%T", "PRinfo_help", client);`

Reply:
ohhh, so thats why i was getting a index error lol, i just didnt understand how, never used it like u are suggesting ty will change it accordingly